packages/loopring_v3/contracts/core/iface/IExchangeV3.sol

```diff
@@ -343,6 +343,53 @@ abstract contract IExchangeV3 is Claimable
         view
         returns (uint96);
 
+
+    /// @dev Flash mints tokens on L2.
+    ///      The amount minted has to be repaid using `repayFlashMint`.
+    ///
+    ///      This function is only callable by the owner.
+    ///
+    /// @param flashMints The list of flash mints to be done.
+    function flashMint(
+        ExchangeData.FlashMint[] calldata flashMints
+        )
+        external
+        virtual;
+
+    /// @dev Repays funds minted using `flashMint`.
+    /// @param from The address that deposits the funds to the exchange
+    /// @param tokenAddress The address of the token, use `0x0` for Ether.
+    /// @param amount The amount of tokens to deposit
+    /// @param extraData Optional extra data used by the deposit contract
+    function repayFlashMint(
```

Comment:
shall we also allow users to "withdrawlToPayFlashMint", this may enable another set of use cases.

Reply:
I don't think it enables any new use case, but it could be a bit more efficient in case the flash mints are used for L2 only operations (otherwise some token conversion needs to happen on L1 and a different token is withdrawn than is used for repayment). If we would offer users access to flash minting I would assume all of them would simply pass through an account owned by us because that way we can batch them together, and then we can also just use the existing methods to repay it.

I would say we can add it we ever would do that if we want to save gas in that specific case (presumably, don't know if it will save much gas), but not add features on critical paths until we plan on using them.